metricbeat/module/aerospike/namespace/namespace.go

```diff
@@ -105,7 +112,7 @@ func (m *MetricSet) Fetch(reporter mb.ReporterV2) error {
 // create an aerospike client if it doesn't exist yet
 func (m *MetricSet) connect() error {
```

Comment:
I wonder if it's worth creating a small integration test for this method to check if SSL connection is successful. WDYT?  @shmsr

Reply:
Yes. If it is simple to add, +1 to this.

Reply:
Hi,

In this comment https://github.com/elastic/beats/pull/38126#issuecomment-1999489894 I added the docker-compose and TLS configuration I used to test TLS. Let me know if you have any question.

The only problem I see is that the original test/expected version of the server is 3.9.0, which is very old and I am not sure it supports TLS/it is worth the hassle of finding out how to configure it (we may use the NGINX version I provided in the original comment to avoid relying on aerospike-version).

BTW Using a modern version like `AEROSPIKE_VERSION=ee-6.4.0.7_2` (ee = enterprise-edition) should do the trick.

Reply:
Yeah, let's update it. Anyway, the current version 3.9 is very old (you even mentioned).